complete/complete.go

```diff
@@ -249,7 +249,7 @@ func s3predictor(a cmp.Args) []string {
 		url.Prefix = s3key
 
 		for object := range client.List(ctx, url, true, s3MaxKeys) {
-			if object.IsMarker() {
+			if object.Err != nil {
```

Comment:
let's add a `// TODO` marker to log the error once we have a better logging mechanism.